libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -197,4 +201,51 @@
   .feds-promo--dark a:hover {
     text-decoration: underline;
   }
+
+
+  .feds-links-strip-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
```

Comment:
This probably has no effect, since it doesn't track its parent's height in any way.

Reply:
True, removed it